models/food.js

```diff
@@ -1,11 +1,47 @@
 'use strict';
+var pry = require('pryjs');
```

Comment:
If it works for you, I'm going to go ahead and remove this pry to avoid the heroku error.